app/views/shared/forms/vehicles/_estimated_value.html.erb

```diff
@@ -9,7 +9,6 @@
     <%= form.govuk_text_field(
           :estimated_value,
           hint: t('.use_car_valuation_sites'),
-          label: nil,
```

Comment:
`label: nil` means that we don't want to have a label for that input field. In that case it's because the title acts as a label : 
![image](https://user-images.githubusercontent.com/482806/62524357-6344c080-b82d-11e9-9ba1-1441aa836fc6.png)

If you remove `label: nil`, The form builder inserts a label and Rails makes up the text based on the attribute (`estimated_value` becomes "Estimated value") :
![image](https://user-images.githubusercontent.com/482806/62524337-5922c200-b82d-11e9-8df3-e0f019ec0d76.png)

Reply:
Oh damn!  

This was removed because it was failing Webhint aXe test:
> Form elements should have a visible label
> Form elements must have labels

Reply:
I think this does it:
```ruby
  <%= govuk_form_group show_error_if: @form.errors.present? do %>
    <%= govuk_fieldset_header do %>
      <%= label_tag(:estimated_value, page_title) %>
    <% end %>

    <%= form.govuk_text_field(
          :estimated_value,
          hint: t('.use_car_valuation_sites'),
          label: nil,
          value: number_to_currency_or_original_string(@form.model.estimated_value),
          input_prefix: t('currency.gbp'),
          class: 'govuk-!-width-one-third'
        ) %>
  <% end %>
```

Reply:
This removes the `<h1>` and so fails for a different reason.